recipes/default.rb

```diff
@@ -17,11 +17,9 @@
 # limitations under the License.
 #
 
-include_recipe 'xml::ruby'
-
-chef_gem "fog" do
+chef_gem 'aws-sdk' do
   action :install
-  version node['route53']['fog_version']
+  compile_time false if Chef::Resource::ChefGem.method_defined?(:compile_time)
```

Comment:
I don't think it'll work on all versions of Chef, see: https://github.com/opscode-cookbooks/aws/pull/110

Prefer:
```ruby
compile_time true if Chef::Resource::ChefGem.instance_methods(false).
                                                                            include?(:compile_time)
```